LINK/usr/bin/appliance_console

```diff
@@ -1,7 +1,10 @@
-#!/bin/bash
-# description: ManageIQ appliance console launcher
-#
+#!/usr/bin/env ruby
 
-[[ -s /etc/default/evm ]] && source /etc/default/evm
+require "bundler"
+Bundler.setup
 
-ruby /var/www/miq/vmdb/gems/pending/appliance_console.rb $@
+require "pathname"
+RAILS_ROOT = Pathname.new("/var/www/miq/vmdb")
+
+require "manageiq-gems-pending"
+require "appliance_console"
```

Comment:
@carbonin  I think this file should live with appliance_console itself, maybe?  Then, /usr/bin/appliance_console could simply be a call to run that file?